modules/ols-enabling-postgresql-persistence.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-postgresql-persistence_{context}"]
+= Enabling PostgreSQL persistence
+
+Modify the `OLSConfig` custom resource (CR) file to enable PostgreSQL persistence for the {ols-long} service.
+
+.Prerequisites
+
+* You are logged in to the {ocp-product-title} web console as a user account that has permission to create a cluster-scoped CR file, such as a user with the `cluster-admin` role.
+
+* You have installed the {ols-long} Operator.
+
+* You have configured the LLM provider.
+
+.Procedure 
+
+. In the {ocp-product-title} web console, click *Operators* -> *Installed Operators*. 
+
+. Select *All Projects* in the  *Project* dropdown at the top of the screen.
+
+. Click *{ols-long} Operator*.
+
+. Click *OLSConfig*, then click the `cluster` configuration instance in the list.
+
+. Click the *YAML* tab.
+
+. Insert the `spec.ols.rag` yaml code:
++
+.Example `OLSconfig` CR file
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: ols.openshift.io/v1alpha1
+kind: OLSConfig
+metadata:
+  name: cluster
+  namespace: openshift-lightspeed
+spec:
+  llm:
+    providers:
+...
+  ols:
+    storage: {} <1>
+----
+<1> If you do not know what values to use, enter empty braces for the `storage` parameter to use the default values.
++
+Alternatively, if specific values are known you can enter them for the `size` and `class` parameters. 
```

Comment:
You can override these defaults with the `size` and `class` parameters:

Reply:
updated based on discussion